Source/Core/Proj4Projection.js

```diff
@@ -0,0 +1,196 @@
+define([
+        './Cartesian3',
+        './Cartographic',
+        './Math',
+        './Check',
+        './defaultValue',
+        './defined',
+        './defineProperties',
+        './Ellipsoid',
+        './oneTimeWarning',
+        './Rectangle',
+        '../ThirdParty/proj4-2.5.0'
+    ], function(
+        Cartesian3,
+        Cartographic,
+        CesiumMath,
+        Check,
+        defaultValue,
+        defined,
+        defineProperties,
+        Ellipsoid,
+        oneTimeWarning,
+        Rectangle,
+        proj4) {
+    'use strict';
+
+    // proj4 may crash with absolute extents
+    var smallerThanMax = Rectangle.fromDegrees(-180.0 + CesiumMath.EPSILON7, -90.0 + CesiumMath.EPSILON7, 180 - CesiumMath.EPSILON7, 90 - CesiumMath.EPSILON7);
+
+    /**
+     * MapProjection using proj4js. This projection is only to be used with Ellipsoid.WGS84.
```

Comment:
> This projection is only to be used with Ellipsoid.WGS84.

Is this a proj4 limitation?  WKT includes spheroid/ellipsoid definition in it's string, what's stopping us from using it?